packages/reports/addon/components/navi-column-config.hbs

```diff
@@ -15,11 +15,13 @@
             @onRemoveColumn={{fn @onRemoveColumn column.fragment}}
             @cloneColumn={{fn this.cloneColumn column}}
             @onAddFilter={{fn this.onAddFilter column.fragment}}
+            @onUpsertSort={{fn @onUpsertSort column.fragment}}
```

Comment:
Do we have a way for users to control the sort order?

Reply:
https://github.com/yavin-dev/framework/issues/1286